compiler/luci-interpreter/src/kernels/Utils.cpp

```diff
@@ -178,5 +178,14 @@ Shape calculateShapeForBroadcast(const Shape &input1_shape, const Shape &input2_
   return output_shape;
 }
 
+void throwIfConditionFalse(bool condition, const char *message)
```

Comment:
Usually this is implemented as a macro, so you don't have to provide the message (it is generated from the code).
It is also easier to print the line number and the file name where the error occured with macro.

Reply:
Okay, I will change it. and If I change it as you said(not message but file and line number), the comment below will be solved!